City-of-Helsinki/berth-reservations-admin
# VEN-550 | Fix the text of applications list's filters

## Description :sparkles:
- Change application filter text from “Vaihto sataman sisällä” to “Vaihtohakemukset”.

## Issues :bug:

### Closes :no_good_woman:

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:
- After you log in, click on `Hakemukset` in the sidebar
- The table's header should show three filters that read: 
KAIKKI, VAIHTOHAKEMUKSET, UUDET HAKEMUKSET

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/78001439-56398c00-733e-11ea-8801-262ba01ccf68.png)

## Additional notes :spiral_notepad:
